libraries/botbuilder-dialogs/src/dialog.ts

```diff
@@ -1,6 +1,3 @@
-/**
- * @module botbuilder-dialogs
- */
```

Comment:
@JonathanFingold do we no longer need this in our files? What's the rule that we on the SDK team should follow for this docstring/comment?

Reply:
@stevengum Our pubs team uses typeDoc + type2docFx + docFx. Only one `@module` comment is needed per module, and then only if it includes a description. (typeDoc is smart enough to figure out the module name on its own.)

Without a description, it is effectively a no-op (for our publishing pipeline). If there are multiple such tags across multiple files within a module. I believe the "first" description encountered is used.

Reply:
we needed this comment do to a TypeDoc plugin we were using. You're saying Docs.com doesn't use the same module plugin? It used to.

Reply:
Here's the plugin: https://github.com/christopherthielen/typedoc-plugin-external-module-name

Reply:
@stevenic - I've asked the publishing team to comment on this thread. (For what it's worth, I've removed the `@module` from other files in previous PRs, [TurnContext](https://docs.microsoft.com/en-us/javascript/api/botbuilder-core/turncontext?view=botbuilder-ts-latest), for instance.)

Reply:
@stevengum @Stevenic - I got confirmation this morning, "We’re not using any plugin for typedoc."